src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
 spring.application.name=spring
+server.port=8080
```

Comment:
Actually Tomcat uses 8080 by default, so in this particluar project you don't need to specify the port.